x/checkers/keeper/msg_server_play_move.go

```diff
@@ -2,16 +2,61 @@ package keeper
 
 import (
 	"context"
+	"errors"
+	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	rules "github.com/xavierlepretre/checkers/x/checkers/rules"
 	"github.com/xavierlepretre/checkers/x/checkers/types"
 )
 
 func (k msgServer) PlayMove(goCtx context.Context, msg *types.MsgPlayMove) (*types.MsgPlayMoveResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	// TODO: Handling the message
-	_ = ctx
+	storedGame, found := k.Keeper.GetStoredGame(ctx, msg.IdValue)
+	if !found {
+		return nil, errors.New("Game not found " + msg.IdValue)
+	}
 
-	return &types.MsgPlayMoveResponse{}, nil
+	// Is it an expected player?
+	var player rules.Player
+	if strings.Compare(storedGame.Red, msg.Creator) == 0 {
```

Comment:
```suggestion
	if strings.Compare(storedGame.Red, msg.Creator) {
```

Reply:
The compiler does not accept this change as this is not boolean.